### PR TITLE
Added TEKTELIC Seal, Memo and Tempo Devices

### PR DESCRIPTION
**Pull request by ssharma-tektelic**
#### Summary
Added TEKTELIC Seal, Memo and Tempo devices.

#### Changes

Modified existing TEKTELIC files to include addition of  Seal, Memo and Tempo devices,  and created new codecs and profiles for these new devices.

#### Checklist for Reviewers

- [ ] Title and description should be descriptive (Not just a serial number for example).
- [ ] `vendorProfileID` should not be `vendorID`.
- [ ] All devices should be listed in the vendor's `index.yaml` file.
- [ ] At least 1 image per device and should be transparent.

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: bin/validate.js [hunk 1]
```diff
@@ -59,7 +59,9 @@ async function requireDimensions(path) {
         reject(new Error(`load image ${path}: ${err}`));
```
Comment (johanstokking) [inline bin/validate.js:59]: @ssharma-tektelic Please revert changes in this file

**Review by johanstokking — COMMENTED** [review]
Thanks! @Jaime-Trinidad or @LDannijs will pick this up. One comment though.

**Review by johanstokking — APPROVED** [review]

**Review by LDannijs — APPROVED** [review]

File: vendor/tektelic/t0006377-memo.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+name: TEKTELIC MEMO Multi-Purpose Display Tablet
```
Comment (LDannijs) [inline vendor/tektelic/t0006377-memo.yaml:1]: I forgot to mention, please don't put the vendor name in the name of the device.
```suggestion
name: MEMO Multi-Purpose Display Tablet
```

File: vendor/tektelic/t0006377-tempo.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
+name: TEKTELIC TEMPO Meeting Room Booking Solution
```
Comment (LDannijs) [inline vendor/tektelic/t0006377-tempo.yaml:1]: The other devices (tempo, seal, breeze-v and stork) still have TEKTELIC in their name too.

**Review by LDannijs — APPROVED** [review]

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**